config/initializers/node-orm2.js

```diff
@@ -19,9 +19,6 @@ export default {
         adapter.db = db;
         let models = container.lookupAll('model');
         let adapterModels = mapValues(models, (Model) => {
-          if (Model.hasOwnProperty('abstract') && Model.abstract) {
-            return;
-          }
```

Comment:
@davewasmer Yep and I liked the idea.  However in https://github.com/denali-js/denali/commit/59ec7e0adc237ec7b7b26eb00a0a9e98ab3e8573 you removed it, so was trying to follow your lead there ... 😀 In the future do you want me questioning your actions a little more loudly? 😉